# This will raise a MergeError
def f(*): pass
